src/lib/db.js

```diff
@@ -64,6 +90,8 @@ function executeScript (filename) {
 
   if (dbType === 'strong-oracle') {
     return executeSQLPlus(filepath)
+  } else if (dbType === 'pg') {
+    return executePSQL(filepath)
```

Comment:
So now `executeStatements` is only used for sqlite? What's the advantage of doing it this way? Is this working around a `knex` issue?